src/components/Colorbox.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+const colors = {
+  items: 28,
+  cssColors: [
+    "Black",
```

Comment:
Where are this colours coming from?

Reply:
I looked the colors in the paint photo and some colors are different because I didn't find the same colors in some cases, but they come from the photo.